src/tool_item_group.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ffi;
+use glib::{Cast, Object};
+use ToolItem;
+use ToolItemGroup;
+use IsA;
+use glib::translate::*;
+use glib::Value;
+use Container;
+use Widget;
+
+impl ToolItemGroup {
+    pub fn set_item_expand<T: IsA<ToolItem> + IsA<Object> + IsA<Widget>>(&self, item: &T, expand: bool) {
+        let object = self.clone();
+        unsafe { ffi::gtk_container_child_set_property(object.upcast::<Container>().to_glib_none().0,
+                                                       item.to_glib_none().0,
+                                                       "expand".to_glib_none().0,
+                                                       Value::from(&expand).to_glib_none().0) }
+    }
+
+    pub fn get_item_expand<T: IsA<ToolItem> + IsA<Object>+ IsA<Widget>>(&self, item: &T) -> bool {
+        let mut value = unsafe { Value::uninitialized() };
```

Comment:
For `i32` this not work, IMHO for `bool` too.
I use `let mut value = 0.to_value();`